core/comment.ts

```diff
@@ -307,8 +296,10 @@ export class Comment extends Icon {
       browserEvents.unbind(this.onInputWrapper_);
       this.onInputWrapper_ = null;
     }
-    this.bubble_.dispose();
-    this.bubble_ = null;
+    if (this.bubble_) {
+      this.bubble_.dispose();
+      this.bubble_ = null;
```

Comment:
Why do you still need to set `this.bubble_` to `null` here?

Reply:
This is just disposing of the whole bubble, not the icon. So we need to make sure to drop references to the bubble so it can be disposed.